### PR TITLE
Move com.ibm.ws.jaxrs.2.0_fat test libs to build/lib/autoFVT/appLibs

### PR DESCRIPTION
**Pull request by WhiteCat22**
#18202 

Move test application libs from `build/lib/autoFVT/publish/shared/resources` to `build/lib/autoFVT/appLibs` so they are not inadvertently added to the test harness's classpath. (Test harness adds everything under `autoFVT/publish/**.*jar`

https://github.com/OpenLiberty/open-liberty/blob/integration/dev/fattest.simplicity/autoFVT-defaults/src/ant/launch.xml#L269

### CODE REVIEW

**Comment by WhiteCat22** [comment]
#build

**Review by jim-krueger — COMMENTED** [review]
There are a few files missing copyright updates, and I have a questions:  Can any of the permission additions be fixed in our code or in the testcase? (just asking)
Other than that it looks good to me.

File: dev/com.ibm.ws.jaxrs.2.0_fat/fat/src/com/ibm/ws/jaxrs20/fat/ExtraProvidersTest.java [hunk 1]
```diff
@@ -31,7 +30,7 @@
 public class ExtraProvidersTest extends AbstractTest {
 
     // Use the same server as JAXRSWebContainerTest
-    @Server("com.ibm.ws.jaxrs.fat.webcontainer")
+    @Server("com.ibm.ws.jaxrs.fat.extraproviders")
```
Comment (andymc12) [inline dev/com.ibm.ws.jaxrs.2.0_fat/fat/src/com/ibm/ws/jaxrs20/fat/ExtraProvidersTest.java:33]: nice catch!

File: dev/com.ibm.ws.jaxrs.2.0_fat/fat/src/com/ibm/ws/jaxrs20/fat/UTF8Test.java [hunk 2]
```diff
@@ -1,3 +1,4 @@
+package com.ibm.ws.jaxrs20.fat;
```
Comment (andymc12) [inline dev/com.ibm.ws.jaxrs.2.0_fat/fat/src/com/ibm/ws/jaxrs20/fat/UTF8Test.java]:
```suggestion
```

File: dev/com.ibm.ws.jaxrs.2.0_fat/fat/src/com/ibm/ws/jaxrs20/fat/UTF8Test.java [hunk 3]
```diff
@@ -8,7 +9,7 @@
  * Contributors:
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
-package com.ibm.ws.jaxrs20.fat.json;
+
```
Comment (andymc12) [inline dev/com.ibm.ws.jaxrs.2.0_fat/fat/src/com/ibm/ws/jaxrs20/fat/UTF8Test.java]:
```suggestion
package com.ibm.ws.jaxrs20.fat;
```

**Comment by WhiteCat22** [comment]
@jim-krueger I'm not entirely sure, I happened to run the same tests locally with Java 8 in addition to Java 11 and they were failing because of these permissions. Not sure why, my changes shouldn't affected them. But, I decided to "fix" them while I was looking at them.

If you want to open an issue, we can investigate those permissions further in the future ;)

**Review by andymc12 — APPROVED** [review]
I like the move from separate packages into the main package - that will be a huge improvement.  Thanks!  As far as copyrights go, there was a huge debate in slack on this last week... I'm not sure it came to a conclusion, but it sounds like small or insignificant changes do not warrant a change to the copyright year.  That's pretty vague, but I would say that changing package names or directories for libraries are insignificant.  The only file that I think should be updated to 2021 is the `build.gradle` since it restructures where libraries will end up. I also made a suggestion in the UTF8Test where the package line got put above the copyright header.  Other than these changes, everything looks good.  Thanks again!

**Comment by WhiteCat22** [comment]
I did the copyright statements for all of the test classes I moved. I'm away of the "% changed" rule for copyright statements and decided that a package move could go either way so I just did it.

**Comment by WhiteCat22** [comment]
#run-libby-bot

**Pull request merged**